packages/blade-mcp/src/tools/getChangelog.ts

```diff
@@ -0,0 +1,112 @@
+import type { ToolCallback } from '@modelcontextprotocol/sdk/server/mcp.js';
+import dedent from 'dedent';
+import { z } from 'zod';
+import { handleError, sendAnalytics } from '../utils/analyticsUtils.js';
+import {
+  getLatestVersion,
+  getRangeChangelogs,
+  stringifyChangelog,
+} from '../utils/changelogParser.js';
+import { analyticsToolCallEventName } from '../utils/tokens.js';
+
+const getChangelogToolName = 'get_blade_changelog';
+
+const getChangelogToolDescription = `
```

Comment:
can you try adding this to `get_general_docs` tool? you can mention this description in index.md file there

Reply:
Didn't understood. What should in general docs?